Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -748,7 +775,33 @@ public static void Shutdown()
 
         #region Server Internal Message Handlers
 
-        void OnServerConnectInternal(NetworkConnection conn, ConnectMessage connectMsg)
+        void OnServerAuthenticateInternal(NetworkConnection conn, ConnectMessage connectMsg)
```

Comment:
rename to OnServerConnectInternal

Reply:
There already is a OnServerConnectInternal below that a bit

Reply:
Right,  and I am saying:  get rid of it.
move all the code that exists in OnServerConnectInernal  into OnServerAuthenticated